itez/beneficiary/models.py

```diff
@@ -41,3 +41,31 @@ class District(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class WorkDetail(models.Model):
+    """
+    Include Work Detail properties. 
+    """
+    gross_pay = models.DecimalField( 
+        _("Monthly Salary"),
+        decimal_places=2,
+        max_digits=1000,
+        null=False
+    )
+    company = models.CharField(
+        _("Company Name"),
+        max_length=200,
+        null=False
+    )
+    insured = models.BooleanField(
+        _("Company Insured"),
+        default = False
+    )
+    work_address = models.CharField(
```

Comment:
let this be a textfield 
`work_address=models.TextField(_(Work Address), null=True, blank=True)`

Reply:
then ensure to update migrations after